src/apps/MASS3DEA-Hip.cpp

```diff
@@ -99,72 +110,71 @@ void MASS3DEA::runHipVariantImpl(VariantID vid) {
 
     using inner_z = RAJA::LoopPolicy<RAJA::hip_thread_size_z_loop<MEA_D1D>>;
 
+    using inner_zyx = RAJA::LoopPolicy<RAJA::hip_thread_zyx_loop>;
+
+    using inner_yxz = RAJA::LoopPolicy<RAJA::hip_thread_yxz_loop>;
+
     startTimer();
     for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
 
+#if 1
```

Comment:
@artv3 do you plan to keep both options in the code? If so, I recommend using a descriptive macro definition name here instead of just '1' or '0' like we do here: https://github.com/LLNL/RAJAPerf/blob/develop/src/basic/NESTED_INIT-OMP.cpp#L43